reactor/db/models/signals.py

```diff
@@ -0,0 +1,50 @@
+from functools import partial
+
+from django.db.models import signals
+
+DEFAULT_RECEIVERS = {
+    "pre_init": signals.pre_init,
+    "post_init": signals.post_init,
+    "pre_save": signals.pre_save,
+    "post_save": signals.post_save,
+    "pre_delete": signals.pre_delete,
+    "post_delete": signals.post_delete,
+    "m2m_changed": signals.m2m_changed,
+    "pre_migrate": signals.pre_migrate,
+    "post_migrate": signals.post_migrate,
+}
+
+
+def _get_receiver(name):
+    def _receiver(name, sender, **kwargs):
+        return getattr(kwargs.get("instance") or sender, name)(**kwargs)
+
+    return partial(_receiver, name)
+
+
+def connect_signals(model):
+    """Connects signals to receivers defined as the model's methods.
```

Comment:
Use imperative mood in docstrings.

```suggestion
    """Connect signals to receivers defined as the model's methods.
```